lib/src/contact/contact_details.dart

```diff
@@ -142,12 +142,13 @@ class _ContactDetailsState extends State<ContactDetails> with ChatCreateMixin {
                     iconBackground: CustomTheme.of(context).chatIcon,
                     onTap: () => createChatFromContact(context, widget.contactId),
                   ),
-                  SettingsItem(
-                    icon: IconSource.flag,
-                    text: L10n.get(L.settingItemFlaggedTitle),
-                    iconBackground: CustomTheme.of(context).flagIcon,
-                    onTap: () => _settingsItemTapped(context, SettingsItemName.flagged),
-                  ),
+//TODO: This should be discussed (Flagged messages per user)!
+//                  SettingsItem(
+//                    icon: IconSource.flag,
+//                    text: L10n.get(L.settingItemFlaggedTitle),
+//                    iconBackground: CustomTheme.of(context).flagIcon,
+//                    onTap: () => _settingsItemTapped(context, SettingsItemName.flagged),
+//                  ),
```

Comment:
Was it a request from the design team? If not, we should discuss this now before we merge it. Just my thoughts :)

Reply:
The designs say that a contact profile has a flagged messages item.

Reply:
The decision to take it out for the moment was lined up with @Boehrsi yesterday.

Reply:
Alright